index.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express');
 const path = require('path');
-
+const queries = require('./src/queries/index');
```

Comment:
you don't need to add the word `index` in your path, when node goes through the `require` if it sees a path like this `/folder/new-folder` then it will automatically try and get the index file
so `./src/queries` is enough